src/index.ts

```diff
@@ -77,11 +78,27 @@ export type ChannelHandlersById = {
 
 const channelHandlersById: ChannelHandlersById = {};
 
-const addHandler = (channelId: string, channelHandlers: ChannelHandlers) => {
-  channelHandlersById[channelId] = [
-    ...(channelHandlersById[channelId] || []),
-    channelHandlers,
-  ];
+const addHandler = (
+  oneOrMoreChannels: string | string[],
+  oneOrMoreHandlers: ChannelHandlers | ChannelHandlers[],
+) => {
+  const channels =
```

Comment:
`const channels = [].concat(oneOrMoreChannels);` is an easy way to do this btw

Reply:
TS complains, at least
![Uploading Screen Shot 2021-12-31 at 7.32.07 PM.png…]()

Reply:
TS is stupid